### PR TITLE
New circus clown outfit!!!

### PR DESCRIPTION
**Pull request by ATDoop**
A new outfit for clowns, the circus clown outfit!! This includes a clown nose and neck ruffle. This also edits clown loadouts to allow masks as an option, but visitors still spawn with masks so I think it's fine? Also adds a speciesId to decapoids and grays, so you can make specific clothing variants for them now!

I'm not sure if there's anything I'm forgetting, and hopefully this doesn't break anything... I tested stuff to the best of my abilities but touching loadouts is pretty scary!!

The offset for the hats is _slightly_ off. I have no idea how to fix this, I've fiddled with the values so much and couldn't figure it out. It's not very noticeable in game at the default zoom value, at least. Also, the gray hat is a bit fucked up, I will fix that later. (edit: nevermind i couldn't figure it out fuuuck!!)
image

:cl:
- add: Added a new clown outfit! Includes a neck ruffle and clown nose, have fun mixing and matching!!

### CODE REVIEW

File: Resources/Prototypes/_Impstation/Entities/Mobs/Species/decapoid.yml [hunk 1]
```diff
@@ -155,6 +155,7 @@
     needsHands: false # they drag with the claw, or their back legs
     applySpeedModifier: false # they're strong as fuck. also i made this variable just for them.
   - type: Inventory
+    speciesId: gray
```
Comment (Darkmajia) [inline Resources/Prototypes/_Impstation/Entities/Mobs/Species/decapoid.yml:158]: were decapoids supposed to share a species id with grays? thats scary..

File: Resources/Prototypes/Roles/Jobs/Civilian/clown.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@
 - type: startingGear
   id: ClownGear
   equipment:
-    mask: ClothingMaskClown
+    #mask: ClothingMaskClown #imp change, i don't know a better way to do this. clown visitors seem to still spawn with masks so i think it's fine.
```
Comment (Darkmajia) [inline Resources/Prototypes/Roles/Jobs/Civilian/clown.yml:31]: yeah this is totally fine, the whole ling can get deleted

**Review by Darkmajia — COMMENTED** [review]
i love this so much

File: Resources/Prototypes/_Impstation/Entities/Mobs/Species/decapoid.yml (hunk 1 above)
Comment (ATDoop) [reply Resources/Prototypes/_Impstation/Entities/Mobs/Species/decapoid.yml:158]: OOPS this explains a lot. reason why you never make one more change after telling yourself to take a break

**Review by Darkmajia — APPROVED** [review]

**Pull request merged**